src/components/AddTodoForm/AddTodoForm.tsx

```diff
@@ -0,0 +1,116 @@
+import { FormEvent, useState } from 'react';
+import { AddTodoFormValues, Todo, User } from '../../types/types';
+import {
+  DEFAULT_VALUES,
+  MISSING_TITLE_ERROR,
+  MISSING_USER_ERROR,
+} from './constants';
+
+type Props = {
+  users: User[];
+  nextId: number;
+  onAddTodo: (todo: Todo) => void;
+};
+
+export const AddTodoForm: React.FC<Props> = ({ users, nextId, onAddTodo }) => {
+  const [formValues, setFormValues] =
+    useState<AddTodoFormValues>(DEFAULT_VALUES);
+  const [titleError, setTitleError] = useState<null | string>(null);
```

Comment:
u can use enum for ur error. try it
will be - `''` by default
```suggestion
  const [titleError, setTitleError] = useState<ErrorType>(ErrorType.Default);
```